src/Card/CardStatus.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from '..';
+
+const CardStatus = React.forwardRef(({
+  className,
+  children,
+  variant,
+  icon,
+}, ref) => (
+  <div
+    className={classNames('pgn__card-status',
+      `alert-${variant}`, 'alert', 'alert-content',
```

Comment:
[curious] Which color variant should a consumer use for the darker `Card.Status` variant (primary-500 with white text)? If we're simply using the `alert` classes here, we're still limited by how it's styled, which is using the `*-100` color level for each variant (e.g., primary-100) so as far as I can tell, there's no way to get this color variant? 

<img width="404" alt="image" src="https://user-images.githubusercontent.com/2828721/181752253-e8118df0-a074-481a-bc0f-da0aa70d9464.png">

Reply:
Also related to the above screenshot, how should a consumer implement a heading and content inside of the `children` for this component?

If the API were similar to `Alert`, there might be a `Card.Status.Heading` component and allow something like:

```jsx
<Card.Status icon={Check} variant="success">
  <Card.Status.Heading>Locked lorem ipsum dolor sit amet</Card.Status.Heading>
  <p>
    Review your grades below for more details.
  </p>
</Card.Status>
```

However, it seems this would result in the following behavior currently:

<img width="971" alt="image" src="https://user-images.githubusercontent.com/2828721/181753012-d7c75ca3-e478-44de-93bd-4561ddc6d721.png">

Reply:
For the first comment: I overridden `primary` variant using css only for `Card.Status` (`primary-500` background and white text)
For the second comment: I added `title` prop that is similar to `Card.Section` `title` prop

Reply:
This works as well :)